### DISCUSSION TITLE
RGB Status LED

### DISCUSSION THREAD

**Discussion by crazy-hair** [suchmememanyskill/CYD-Klipper#73] [Ideas]
With the 3.5" LCD, the LED is facing the front, which would make prime real estate to convey printer status with the screen off (maybe green for printing, blue for idle, and red for error state?). That way you'd be able to take a quick glance at the display and see if everything is still going as it should without turning the display back on.

**Comment by smokeandlights** [comment]
Honestly, I printed the back of my 2.8" case in transparent, and I think it would still work quite well for this, too.
